src/scancode/help.py

```diff
@@ -24,6 +24,11 @@
 
     scancode -lc --package --ignore README --processes 4 --json-pp - samples/
 
+Scan a file for copyright and packages. Print scan results to file as pretty
+formatted JSON.
+
+    scancode --copyright --package  samples/zlib/zlib.h --json-pp - > scan.json
```

Comment:
Why would you promote the the use of a stdout output with a redirection to a file?
Also this kind of example might be best in the doc than in the cli... @AyanSinhaMahapatra what do you think?

Reply:
IMHO if the scan target is only a file or two then using stdout output makes sense, otherwise, file output works best(your opinions?). Are you asking about having examples only in the doc and not CLI(minimal examples), if so then yes that would be best. Do you suggest any changes to the current doc then?

Reply:
> IMHO if the scan target is only a file or two then using stdout output makes sense, otherwise, file output works best(your opinions?). Are you asking about having examples only in the doc and not CLI(minimal examples), if so then yes that would be best. Do you suggest any changes to the current doc then?

@AyanSinhaMahapatra I am not suggesting any change. Your call if you want to integrate something like that in the doc. Otherwise I would like this PR closed if we do not merge it.


as for the stdout... yes output to `-` is fine what I find dubious would be to recommend to use `>` a redirection to a file as a standard example (this is always possible but not the recommended default way IMHO)